ChangeLog.d/move_internal_headers.txt

```diff
@@ -0,0 +1,6 @@
+API changes
+   * Move internal headers.
+     All internal headers have been moved to library/ to unify them in one
+     location that is separate from the public API. This includes some files
+     that were previously not explicitly internal such as: `bn_mul.h`,
+     `aesni.h`, `padlock.h` and `entropy_poll.h`.
```

Comment:
“Move internal headers” is meaningless on its own. Remember that the audience of the changelog is user of the library. They don't know or care about our long-term plans for the library, our motivation for the change, or the context in which the change is made.

Suggested text:

> Header files that were only meant for the library's internal use and were not meant to be used in application code have been moved out of the include/ directory. The headers concerned are bn_mul.h, aesni.h, padlock.h, entropy_poll.h and *_internal.h.

(Also: this is plain text, not markdown.)

Reply:
Done along with another changelog entry for alt implementors.